pages/cheat_sheets/sequencing.md

```diff
@@ -118,13 +120,11 @@ https://www.forskningsetikk.no/en/guidelines/science-and-technology/guidelines-f
 <!--Add one line description-->
 
 #### RDM Services
-- [ELIXIR helpdesk](https://elixir.no/helpdesk)
-  - Provides assistance with:
-    - Data management planning
-    - Storage and computing
-    - Metadata standards
-    - Data deposition to [ELIXIR databases](https://elixir-europe.org/platforms/data/elixir-deposition-databases)
-  - Contact: ELIXIR helpdesk
+[ELIXIR Norway's HelpDesk](https://elixir.no/helpdesk) can assist you with guidance on:
```

Comment:
I would say we are providing much more than "guidance".

Reply:
True. Maybe we remove 'guidance on' or add something like 'guidance, support, and practical/important tips on:'?

Reply:
Yes, just remove "guidance on" and it works!